src/index.html

```diff
@@ -6,9 +6,57 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="list">
+
+        <ul class="navigation>
+          <li class="nav-item is-active">
+            <a href="#" class="is-active">Apple</a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="nav-link">Samsung</a>
+          </li>
```

Comment:
1. no needs to add an empty line between parent tag and child one
2. name classes following BEM methodology
3. `is-active` selector should only add additional css rules, not repeat `nav-link` rules
4. `li` shouldn't have `is-active`. Only first link
```suggestion
      <nav class="nav">
        <ul class="nav__list>
          <li class="nav__item">
            <a href="#" class="nav__item is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="#" class="nav__link">Samsung</a>
          </li>
```